view/adminhtml/templates/subscription_view.phtml

```diff
@@ -0,0 +1,156 @@
+<?php /** @var Razorpay\Subscription\Block\View $block */ ?>
+
+<?php
+$subscriptionInfo = $block->getSubscrib();
+$invoices = $block->getSubscriptionInvoice();
+?>
+<div class="page-main-actions">
+    <div class="page-actions" data-ui-id="page-actions-toolbar-content-header">
+        <div class="page-actions-inner" data-title="">
+            <div class="page-actions-buttons">
+                <button id="back" title="Back" type="button" class="action-default scalable back" onclick="location.href = '<?php echo $block->getUrl('*'); ?>'">
+                    <span>Back</span>
+                </button>
```

Comment:
add pause resume cancel action next to back button with logic (if active show pause and cancel and not show resume and cancel)

Reply:
already added based on subscription status only